array_analyzer/transform/array_generation.py

```diff
@@ -88,6 +88,8 @@ def get_spot_intensity(coords, im, background, params, search_range=2):
     # make spot size always odd
     spot_size = 2 * int(0.3 * spot_width / pix_size) + 1
     bbox_width = bbox_height = spot_size
+    # Strel disk size for spot segmentation
+    disk_size = int(np.rint(spot_size / 2.5))
```

Comment:
Yes the filter size needs to change with the spot size... thanks for fixing this